From be60c923c5e3953b1967e79b0072791aefffaf59 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 16 May 2017 23:24:45 +0200
Subject: [PATCH 1/1] odroid-c2: sd_fusing.sh: avoid truncating image

When using sd_fusing.sh to write to a file base image the
image should not be truncated.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 sd_fuse/sd_fusing.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/sd_fuse/sd_fusing.sh b/sd_fuse/sd_fusing.sh
index 538185bd17..038f67ba91 100755
--- a/sd_fuse/sd_fusing.sh
+++ b/sd_fuse/sd_fusing.sh
@@ -24,9 +24,9 @@ if [ ! -f $UBOOT ]; then
         exit 1
 fi
 
-sudo dd if=$BL1 of=$1 conv=fsync bs=1 count=442
-sudo dd if=$BL1 of=$1 conv=fsync bs=512 skip=1 seek=1
-sudo dd if=$UBOOT of=$1 conv=fsync bs=512 seek=97
+sudo dd if=$BL1 of=$1 conv=fsync,notrunc bs=1 count=442
+sudo dd if=$BL1 of=$1 conv=fsync,notrunc bs=512 skip=1 seek=1
+sudo dd if=$UBOOT of=$1 conv=fsync,notrunc bs=512 seek=97
 
 sync
 
-- 
2.11.0

